daisy_workflows/sbom_validation/test_sbom_export.sh

```diff
@@ -26,14 +28,18 @@ else
   echo "SBOMTesting: non-empty SBOM file found"
 fi
 
-# Check that the disk export succeeded
-gsutil -q stat $GCS_PATH_OUTDISK
-status=$?
-if [[ $status -eq 0 ]]; then
-  echo "SBOMTesting: Disk tar file successfully found"
+# Check that the disk export succeeded, if the flag is not false
+if [[ $CHECK_DISK_TAR == "false" ]]; then
+  echo "SBOMTesting: skipping check for Disk tar file"
 else
-  echo "SBOMFailed: Disk tar file not found"
-  exit 1
+  gsutil -q stat $GCS_PATH_OUTDISK
```

Comment:
I understand the idea but do we really want to validate the disk file? i so we are not actually validating the SBOM right?

Reply:
The presubmit was originally written for linux, and I was hoping to check that when generating the SBOM, it did not cause the disk file export to fail.

For windows, the bootstrap script does not export a disk, but it exports an SBOM. The disk file is likely exported in the post_install script. The thinking was that checking for a disk on windows would be good, but it would make the presubmit take too long because it would require running post_install as well.

Reply:
Sounds good.